content/docs/distribution/requisitions.md

```diff
@@ -78,27 +78,42 @@ You will then see the detail view of the requisition:
 
 ![Requisition Detail](/docs/distribution/images/requisition-detail.png)
 
+| Column        | Description                                                                                                                                                            |
+| :------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Code**      | Code of the item                                                                                                                                                       |
+| **Name**      | Name of the item                                                                                                                                                       |
+| **Unit**      | The unit of measure for the item                                                                                                                                       |
+| **Our SOH**   | How much stock is available in this store                                                                                                                              |
+| **Their SOH** | How much stock is available in the customer's store                                                                                                                    |
+| **Requested** | The quantity requested by customer                                                                                                                                     |
+| **Supply**    | The quantity that will be supplied by this store                                                                                                                       |
+| **Issued**    | The quantity that has already been issued in `Outbound Shipments`                                                                                                      |
+| **Remaining** | The quantity remaining to be supplied to the customer which will be re-calculated after an `Outbound Shipment` has been created. The calculation is 'Supply - Issued'. |
```

Comment:
Or maybe it's here? 
```suggestion
| **Remaining** | The quantity remaining to be supplied to the customer, from what your store has intended to supply. The calculation is 'Supply - Issued'. It will be re-calculated after an `Outbound Shipment` has been created.  |
```

Reply:
Feels like Supply - Issued already says `from what your store planned to supply` 🤔